src/common/config_opts.h

```diff
@@ -1161,7 +1161,8 @@ OPTION(bluestore_cache_trim_max_skip_pinned, OPT_U32, 64) // skip this many onod
 OPTION(bluestore_cache_type, OPT_STR, "2q")   // lru, 2q
 OPTION(bluestore_2q_cache_kin_ratio, OPT_DOUBLE, .5)    // kin page slot size / max page slot size
 OPTION(bluestore_2q_cache_kout_ratio, OPT_DOUBLE, .5)   // number of kout page slot / total number of page slot
-OPTION(bluestore_cache_size, OPT_U64, 3*1024*1024*1024)
+OPTION(bluestore_cache_size_hdd, OPT_U64, 1*1024*1024*1024)
```

Comment:
Perhaps it makes sense to have generic bluestore_cache_size parameter set to 0 by default as well. We tend to have this for other parameters with hdd/sdd specifics , e.g. bluestore_max_blob_size, bluestore_compression_min_blob_size, bluestore_prefer_deferred_size etc